python/lsst/ts/electrometer/config_schema.py

```diff
@@ -88,6 +88,9 @@
   temperature_attached:
     description: Is the temperature probe attached?
     type: boolean
+  image_service_url:
```

Comment:
I think you need to change the version of the config no? I know this will add some overhead but it is probably the right thing to do.